src/code_80005FD0.c

```diff
@@ -4874,15 +4873,15 @@ void func_80013854(Player *player) {
     }
 }
 
-void func_800139E4(f32 arg0, f32 arg1, s32 arg2, s32 arg3, VehicleStuff *vehicle, TrackWaypoint *waypointList) {
+void determine_toad_turnpike_vehicle_speed(f32 speedA, f32 speedB, s32 arg2, s32 arg3, VehicleStuff *vehicleList, TrackWaypoint *waypointList) {
     VehicleStuff *veh;
```

Comment:
arg2  = numVehicles.

initialize_toad_turnpike_vehicle?

Reply:
the thing who are weird with that are that vehicle for toad turnpike are also init in init_course_vehicles with init_course_vehicles where in it they also define it position

Reply:
`init_course_vehicles` --> `spawn_course_vehicles` or `spawn_vehicles`

`determine_toad_turnpike_vehicle_speed` --> `initialize_toad_turnpike_vehicle` or similar.